apps/charterafrica/src/components/Tool/Contributors.js

```diff
@@ -16,7 +17,13 @@ const Contributors = React.forwardRef(function Contributors(props, ref) {
         </RichTypography>
         <AvatarGroup sx={{ justifyContent: { xs: "center", sm: "right" } }}>
           {list.map((item) => (
-            <Avatar key={item.id} alt={item.name} src={item.avatarUrl} />
+            <Avatar
+              key={item.id}
+              component={Link}
+              href={item.link?.href}
```

Comment:
If we're using Link we need to make sure there is **always** `link.href`. Otherwise, Next.js will throw errors in the console.

Reply:
Can never be `null` or `undefined` since we are appending this at `processPageTools`

Reply:
Then don't use `link?.href` @koechkevin, it implies `link` can be `undefined`/`null`. Just `link.href`.